experimental/packages/opentelemetry-instrumentation-http/test/functionals/http-enable.test.ts

```diff
@@ -46,9 +49,12 @@ import {
   SEMATTRS_HTTP_METHOD,
   SEMATTRS_HTTP_RESPONSE_CONTENT_LENGTH_UNCOMPRESSED,
   SEMATTRS_HTTP_ROUTE,
+  SEMATTRS_HTTP_SCHEME,
```

Comment:
Actually, hm. I wonder if we should use the new ATTR_ imports here from incubating. So the ATTR_ stable attributes should still get imported as-is, but the ATTR_ incubating/experimental attributes should be used and imported from semantic-conventions/incubating.

Reply:
On second thought, this is probably right. It feels a bit awkward to have a mix of `SEMATTRS_*` and `ATTRS_*` but one of the reasons to avoid using the new attributes is if anything has changed. For example, we've been advised not to update db attributes because they are close to stability and we may end up with multiple breaking changes before then. Maybe we only use the new `ATTRS_*` if something didn't exist in the previous version

Reply:
Yeah I decided to leave the old attribute names to be sure there isn't a change/regression